src/Plugin/Block/SignupBlock.php

```diff
@@ -13,16 +15,43 @@
  *   category = @Translation("Newsletter Signup"),
  * )
  */
-class SignupBlock extends BlockBase {
+class SignupBlock extends BlockBase implements BlockPluginInterface {
 
+  /**
+   * {@inheritdoc}
+   */
+  public function blockForm($form, FormStateInterface $form_state) {
+    $form = parent::blockForm($form, $form_state);
+
+    $config = $this->getConfiguration();
+
+    $form['form_action'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('Stanford Mailing List Subscribe URL'),
+      '#description' => $this->t('Example: https://stanford.us14.list-manage.com/subscribe/post?u=84e3c1df5a210f020250ee1d6&amp;id=74d8f03797'),
+      '#default_value' => isset($config['form_action']) ? $config['form_action'] : 'https://stanford.us14.list-manage.com/subscribe/post?u=84e3c1df5a210f020250ee1d6&amp;id=74d8f03797',
+    ];
+
+    return $form;
+  }
```

Comment:
@boznik I added a config form so end users can change the subscribe form. I suspect we want this but need confirmation from you.